python/tvm/auto_scheduler/cost_model/xgb_model.py

```diff
@@ -99,7 +99,7 @@ class XGBModel(PythonBasedModel):
     model_file: Optional[str]
         If is not None, save model to this file after every update.
     adapative_training: bool = False
```

Comment:
Looks like this one is also 'adapative' instead of 'adaptive'.

Reply:
Oh, shoot, but the parameter name itself actually has the extra 'a'... Well, sorry for what I thought was just a comment fix which now looks like it fixing spelling within the code.

Reply:
PR updated.